bmap-rs/src/main.rs

```diff
@@ -104,8 +102,6 @@ fn copy(c: Copy) -> Result<()> {
         .create(true)
         .open(c.dest)?;
```

Comment:
Here I think it is already being assumed that the output is going to be a file. Should this be absorbed by the copy library function too and only be executed if the output is actually a file ?